sdk/core/core-client-rest/package.json

```diff
@@ -61,6 +61,7 @@
   "sideEffects": false,
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
+    "@azure/core-paging": "^1.1.1",
```

Comment:
I am not sure if we really want to do this. I think one appeal of core-client-rest is it has a small code size so perhaps we want to keep it this way. Did you consider having paging and lro helpers live somewhere else? core-util or even core-rest-util? cc @xirzec

Reply:
Yeah, that's a good point actually. Maybe the ultimate solution would be to have helper packages `@azure-rest/core-utils-pagining`, `@azure-rest/core-utils-lro`? That way we can fine-tune what dependencies we take when generating SDKs and an SDK that uses paging but not LRO is not forced to pull everything from LRO and vice-versa.